packages/components/src/components/step-group/step-group.stories.ts

```diff
@@ -44,240 +53,160 @@ export default {
+          <p slot="label">Step 1</p>
+        </sd-step>
 
-/**
- * sd-steps are fully accessibile via keyboard.
- */
+        <sd-step size="sm" orientation="horizontal" state="current">
+          <p slot="label">Step 2</p>
+        </sd-step>
 
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">
-      ${generateTemplate({
-        args
-      })}
-    </div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-step-group');
-    await waitUntil(() => el?.querySelector('sd-step')?.shadowRoot?.querySelector('button'));
-
-    el?.querySelector('sd-step')?.shadowRoot?.querySelector('button')!.focus();
-  }
+        <sd-step size="sm" orientation="horizontal" state="disabled">
+          <p slot="label">Step 3</p>
+        </sd-step>
+      </sd-step-group>
+    </div>
+  `
 };
 
 /**
- * This sample shows how to set the active step programmatically.
+ * Use the `orientation` attribute to set the axis of a step-group.
  */
 
-export const SetActiveStep = {
-  parameters: { controls: { include: [] } },
-  name: 'Sample: Set Active Step',
-  render: () => {
-    return html`
-      <sd-step-group id="set-active" size="lg" orientation="horizontal" active-step="0">
-        <sd-step size="lg" orientation="horizontal">
-          <p slot="label">Lorem ipsum dolor sit</p>
+export const Orientation = {
+  name: 'Orientation',
+  render: () => html`
+    <div class="space-y-5">
+      <sd-step-group size="lg" orientation="horizontal" active-step="1">
+        <sd-step size="lg" orientation="horizontal" state="default">
+          <p slot="label">Step 1</p>
         </sd-step>
 
-        <sd-step size="lg" orientation="horizontal">
-          <p slot="label">Exercitation ullamco laboris</p>
+        <sd-step size="lg" orientation="horizontal" state="current">
+          <p slot="label">Step 2</p>
         </sd-step>
 
-        <sd-step size="lg" orientation="horizontal">
-          <p slot="label">Reprehenderit qui in e name</p>
+        <sd-step size="lg" orientation="horizontal" state="disabled">
+          <p slot="label">Step 3</p>
         </sd-step>
       </sd-step-group>
 
-      <sd-button class="w-20 mt-8" size="sm" id="previous">Previous</sd-button>
-      <sd-button class="w-20 mt-8" size="sm" id="next">Next</sd-button>
-
-      <script type="module">
-        const stepGroup = await document.querySelector('sd-step-group#set-active');
-
-        const nextBtn = document.querySelector('sd-button#next');
-        const prevBtn = document.querySelector('sd-button#previous');
-
-        nextBtn.addEventListener('click', () => {
-          stepGroup.setActiveStep(stepGroup.activeStep + 1);
-        });
-
-        prevBtn.addEventListener('click', () => {
-          stepGroup.setActiveStep(stepGroup.activeStep - 1);
-        });
-      </script>
-    `;
-  }
+      <div class="flex gap-12 h-[20vh] pl-[56px]">
```

Comment:
```suggestion
      <div class="flex gap-12 h-[20em] pl-[56px]">
```

The height is not showing propery:
<img width="942" alt="image" src="https://github.com/user-attachments/assets/f3176848-3dda-4438-beb8-4cbbeda14f0a">

Something like 20em is much closer to the figma docs: 
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/75ed98d9-c8b3-4ca3-bb40-c11950e04abd">

Reply:
Updated